README.md

```diff
@@ -202,12 +202,31 @@ Continuing the example above, the following command verifies the CMS file
 `signedfile` and outputs the `outputfile`. Its contents should be identical
 to the original data in `inputfile` above.
 
-    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl cms -verify -CAfile qsc.crt -inform pem -in signedfile -crlfeol -out outputfile -provider oqsprovider -provider default
+    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl cms -verify -CAfile qsc.crt -inform pem -in signedfile -crlfeol -out outputfile -provider-path _build/oqsprov -provider oqsprovider -provider default
 
 Note that it is also possible to build proper QSC certificate chains
 using the standard OpenSSL calls. For sample code see
 [scripts/oqsprovider-certgen.sh](scripts/oqsprovider-certgen.sh).
 
+### Support of `dgst` (and sign)
+
+Also tested to operate OK is the [openssl dgst](https://www.openssl.org/docs/man3.0/man1/openssl-dgst.html)
+command. Sample invocations building on the keys and certificate files in the examples above:
+
+#### Signing
+
+    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl dgst -provider-path _build/oqsprov -provider oqsprovider -provider default -sign qsc.key -out dgstsignfile inputfile
+
+#### Verifying
+
+    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl dgst -provider-path _build/oqsprov -provider oqsprovider -provider default -signature dgstsignfile -verify qsc.pubkey inputfile
+
+The public key can be extracted from the certificate using standard openssl command:
+
+    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl x509 -provider-path _build/oqsprov -provider oqsprovider -provider default -in qsc.crt -pubkey -noout > qsc.pubkey
+
+The `dgst` command is not tested for interoperability with [oqs-openssl111](https://github.com/open-quantum-safe/openssl).
```

Comment:
Re: "not tested for interoperability": Is this just a disclaimer that we haven't tested it? Or indicating that it's actually known to be not interoperable?  The former is fine, but the latter would be less good.

Reply:
It only means to say it's not tested. Someone mentioned that oqs-openssl has a problem with `dgst` without ever giving a concrete sample (https://github.com/open-quantum-safe/openssl/issues/357) and I didn't try to set up interop testing. Do you deem it sensible/necessary?